### PR TITLE
Update items index

### PR DESCRIPTION
**Pull request by taphill**
PR to update items/index page functionality

  - Visitors/users can not see inactive items
  - Admins/merchants can see inactive items
  - Added feature tests

### CODE REVIEW

File: app/views/items/index.html.erb [hunk 1]
```diff
@@ -11,6 +11,8 @@
 
 <section class="grid-container">
   <% @items.each do |item| %>
+    <% next if !item.active? && !(current_admin? || current_merchant?)%>
```
Comment (jdcarey128) [inline app/views/items/index.html.erb:14]: Cool. I don't think I've seen this used before.

**Review by jdcarey128 — APPROVED** [review]
LGTM

**Review by katemorris — APPROVED** [review]
Looks great!

**Review by smcunning — APPROVED** [review]
LGTM! Merging

**Pull request merged**